test/transactions_test.rb

```diff
@@ -114,6 +114,11 @@ def test_raise_command_errors_in_multi_exec
     assert_equal "s1", r.get("foo")
   end
 
+  def test_empty_multi_exec
+    r.multi do |m|
+    end
+  end
+
```

Comment:
Perhaps we could mock the multi / exec to return an error, causing the test to fail if they're actually invoked, as with `test_raise_command_error_when_exec_fails`